### PR TITLE
Mark TC39 regex escaping proposal as reopened

### PR DESCRIPTION
**Pull request by RohanTalip**
#### Summary

Indicate that the TC39 regex escaping proposal has been reopened.

#### Motivation

It seems more accurate to say that the proposal has been reopened.

#### Supporting details

See @ljharb's comment : https://github.com/tc39/proposal-regex-escaping/issues/37#issuecomment-1057187434

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/guide/regular_expressions/index.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ function escapeRegExp(string) {
 The "g" after the regular expression is an option or flag that performs a global search, looking in the whole string and returning all matches.
 It is explained in detail below in [Advanced Searching With Flags](#advanced_searching_with_flags).
 
-_Why isn't this built into JavaScript?_ There is a proposal to add such a function to RegExp, but it was [rejected by TC39.](https://github.com/tc39/proposal-regex-escaping/issues/37)
+_Why isn't this built into JavaScript?_ There is a (reopened) [proposal](https://github.com/tc39/proposal-regex-escaping) to add such a function to RegExp.
```
Comment (ljharb) [inline files/en-us/web/javascript/guide/regular_expressions/index.md:192]:
```suggestion
_Why isn't this built into JavaScript?_ There is a [proposal](https://github.com/tc39/proposal-regex-escaping) to add such a function to RegExp.
```

**Review by ljharb — COMMENTED** [review]
Proposals aren't "closed" or "opened", and i don't think the history is useful here.

File: files/en-us/web/javascript/guide/regular_expressions/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/javascript/guide/regular_expressions/index.md:192]: I read the thread and I think you are right @ljharb:
- TC39 has decided to reconsider the proposal (likely with modification)
- the historical issue was painful to read with a ton of personal attacks that don't bring anything.

Let's commit Jordan's changes and merge this.

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**